### ISSUE TITLE
关于mask问题

### ISSUE THREAD

**Issue by Caishen-git** [caiyuanhao1998/MST#79]
尊敬的作者，您好，我叫蔡有强，来自湖南益阳。
我运行您的代码时遇到一个问题，根据您论文中提到的使用CAVE数据集训练，KAIST中选出10个场景进行测试，所以我将您论文如此处做了微修改，d = random.randint(0, 1)改成了d=0，以确保只使用CAVE数据集
![1](https://github.com/user-attachments/assets/f6cf41fd-73ca-4665-a010-c45bf55ebfa5)
然而，我运行后，显示我的mask有问题，我在您此篇论文的github中未找到mask.mat可以供下载，故使用的是来自其他地方的mask.mat，运行错误如下
![2](https://github.com/user-attachments/assets/17879856-c5da-4f7f-922d-d5a482359fc7)
我不确定是不是用了和您不一样的mask.mat所导致，我的mask.mat文件中是一个256*256大小的mask。
如果您有时间，可以帮我解决一下这个问题嘛，非常感谢您。

**Comment by caiyuanhao1998** [comment]
你好，请使用 [TSA-Net](https://github.com/mengziyi64/TSA-Net) 提供的 mask.mat 来跑我们的代码。如果还是不行，可以替换 [DGSMP](https://github.com/TaoHuang95/DGSMP) 的 mask 来跑。

如果觉得我们的 repo 有用的话，帮我们点点 star 支持一下，感谢

**Comment by Caishen-git** [comment]
非常感谢您的解答，我弄清楚了我出错的原因，我误以为我在做的是real部分的实验，在Simulation处运行是没有问题的
再次感谢您的解答，以及您在github精心准备的这些指导内容，对我的学习有非常大的帮助，感谢为学术界作出的贡献！

**Comment by Caishen-git** [comment]
作者您好，关于您代码class MST(nn.Module):中forward的        
for i, (FeaUpSample, Fution, LeWinBlcok) in enumerate(self.decoder_layers):段中，FeaUpSample（），Fution（），LeWinBlcok这些是在哪里定义的呢，如图
![image](https://github.com/user-attachments/assets/770e19a8-e056-467b-908f-3f89973e896f)
我不能定位到这些东西定义的位置，以至于我不能理解这些东西，我正处于临近毕业的时间，2024.12.13我就要进行中期答辩了，这让我深陷困境，希望您能帮我解答上述问题!
